concept-anchor.md

```diff
@@ -0,0 +1,36 @@
+## Title
+Concept Anchor for discussions with Management
```

Comment:
The titles of other patterns either describe the solution, or the problem.

Some other ideas for titles for this pattern:

* Anchor InnerSource with Agile / DevOps / Lean
* Explain InnerSource to Management by anchoring it in Agile / DevOps / Lean

Is there an umbrella term, that can be used to describe Agile / DevOps / Lean? That would make the title shorter.

However feel free to leave the title as is. Once we have a Solution for this pattern, and some known instances, we would certainly know better how to name this pattern.